### PR TITLE
update 'final' api versions

### PR DESCRIPTION
**Pull request by kwsutter**
Signed-off-by: Kevin Sutter <kwsutter@gmail.com>

Removed the references to Mx or RCx versions of the APIs.  All of the "final" APIs should now be in staging and available.

Also, updated the maven coordinates for Jakarta Authentication.

### CODE REVIEW

**Review by ivargrimstad — APPROVED** [review]

File: pom.xml
Comment (kazumura) [inline pom.xml:56]: still remain RC2 for 'ejb-api.version'

Comment (kwsutter) [reply pom.xml:56]: Nice catch, @kazumura!  Corrected.

**Comment by kwsutter** [comment]
Will wait for @kazumura's re-review before merging...

**Review by kazumura — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**
